flytekit/models/annotation.py

```diff
@@ -0,0 +1,45 @@
+import json as _json
+from typing import Any, Dict
+
+from flyteidl.core import types_pb2 as _types_pb2
+from google.protobuf import json_format as _json_format
+from google.protobuf import struct_pb2 as _struct
+
+
+class TypeAnnotation:
```

Comment:
could we add the kinda dumb but _surprisingly_ helpful there-and-back-again unit test for this class?  https://github.com/flyteorg/flytekit/blob/742f40eb718777887043db3842b076b7f87a8427/tests/flytekit/unit/models/core/test_types.py#L19 as an example

Reply:
sure